backstop/assets/theme.js

```diff
@@ -37,10 +37,12 @@ class ThemeButton extends HTMLElement {
         display: inline-block;
         margin-left: 0.25rem;
       }
-      :host-context(.iui-theme-dark) svg {
+      :host-context(.iui-theme-dark) svg,
```

Comment:
This will not work in Firefox and Safari 🤔 Could we just use `currentColor` instead?

Reply:
fixed in #511